src/main/groovy/wooga/gradle/build/unity/tasks/UnityBuildPlayerTask.groovy

```diff
@@ -125,12 +130,16 @@ class UnityBuildPlayerTask extends AbstractUnityProjectTask {
             setBuildTarget(getBuildPlatform().toLowerCase() as BuildTarget)
         }
 
+        if (versionCode.present) {
+            customArgs += "versionCode=${versionCode.get()};"
+        }
+
         if (toolsVersion.present) {
-            customArgs += "toolsVersion=${toolsVersion.get()}"
+            customArgs += "toolsVersion=${toolsVersion.get()};"
         }
 
         if (commitHash.present) {
-            customArgs += "commitHash=${commitHash.get()}"
+            customArgs += "commitHash=${commitHash.get()};"
```

Comment:
Oh nice, I somehow forgot we have commitHash already, was discussing today that it would be nice to have for CI. It needs to be set via Gradle?

Reply:
It's not passed automatically no. This plugin is not aware of git.